src/clarity-angular/data/datagrid/datagrid-cell.ts

```diff
@@ -14,10 +15,50 @@ import { DatagridHideableColumn } from "./datagrid-hideable-column";
     `,
     host: {
         "[class.datagrid-cell]": "true",
-        "[class.datagrid-cell--hidden]": "hidden"
+        "[class.datagrid-cell--hidden]": "hidden",
+        "[class.datagrid-signpost-trigger]": "hasSignpost"
     }
 })
-export class DatagridCell {
+export class DatagridCell implements AfterContentInit {
+
+    /********
+     * @property hasSignpost
+     *
+     * @type {boolean}
+     *
+     * @description
+     * This flag is set with ngAfterContentInit. When there is a Signpost in the DatagridCell projected content
+     * we need to add the .daattgrid-signpost-trigger class to the cell to preserve the natural row height of 35px.
+     */
+    public hasSignpost: boolean;
+
+    /*********
+     * @property signpost
+     *
+     * @type {Signpost}
+     *
+     * @description
+     * @ContentChild to detect the presence of a Signpost in the projected content. Is used to set the flag hasSignpost.
+     */
+    @ContentChildren(Signpost) signpost: QueryList<Signpost>;
+
+
+    /********
+     * @function ngAfterContentInit
+     *
+     * @description
+     * Angular lifecycle function that uses this.signpost property to set this.hasSignpost boolean.
+     * It also subscribes to the @ContentChildren changes observable in case the signpost(s) in a cell
+     * are used conditionally.
+     *
+     */
+    ngAfterContentInit() {
+        this.hasSignpost = (this.signpost && this.signpost.length > 0) ? true : false;
```

Comment:
The ternary `myBoolean ? true : false` feels a bit overkill. You can just straight up use the boolean. 😉 
(If you're worried about `undefined` instead of `false`, `!!myBoolean` works great to ensure strict boolean type.)